src/bosh_azure_cpi/lib/cloud/azure/restapi/azure_client.rb

```diff
@@ -120,7 +120,7 @@ def rest_api_url(resource_provider, resource_type, resource_group_name: nil, nam
       url
     end
 
-    def parse_name_from_id(id)
+    def _parse_name_from_id(id)
```

Comment:
what is the purpose of renaming it but not making it private?

Reply:
make it private now.